packages-ts/eqlabs-multisig-integration/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@chainlink/starknet-integration-tests-eqlabs-multisig",
```

Comment:
One additional structure change request: please rename these integration folders to start with `integration-` prefix:

- `packages-ts/eqlabs-multisig-integration` -> `packages-ts/integration-eqlabs-multisig`
- `packages-ts/starkgate-integration` -> `packages-ts/integration-starkgate`

Reply:
Done